consensus/service/src/byzantine_ledger/pending_values.rs

```diff
@@ -22,15 +22,16 @@ pub struct PendingValues<TXM: TxManager, MTXM: MintTxManager> {
 
     /// We need to store pending values vec so we can process values
     /// on a first-come first-served basis. However, we want to be able to:
-    /// 1) Efficiently see if we already have a given transaction and ignore
-    /// duplicates 2) Track how long each transaction took to externalize.
+    ///     1. Efficiently see if we already have a given transaction and ignore
+    /// duplicates
+    ///     2. Track how long each transaction took to externalize.
     ///
     /// To accomplish these goals we store, in addition to the queue of pending
     /// values, a map that maps a value to when we first encountered it.
     /// This essentially gives us an ordered HashMap.
     ///
-    /// Note that we only store a timestamp for values that were handed to us
-    /// directly from a client. That behavior is enforced by
+    /// Note that we only store a received time for values that were handed to
```

Comment:
Renamed `timestamp` in this module to `received_time` to avoid the ambiguity with the block timestamp versus the performance logging of the value processing